server.js

```diff
@@ -18,7 +18,7 @@ const modules = [
 const bot = new Discord.Client();
 const handlers = {};
 modules.forEach(module => {
-    const moduleObject = new (require("./modules/" + module + '.js'))();
+    const moduleObject = new (require("./modules/" + module + '.js'))(true);
```

Comment:
Suggestion: default initialize parameter to `true` and pass in `false` when running tests. I'd rather have the production call be without a parameter and the test call have the extra parameter.